functests/must-gather/must_gather_test.go

```diff
@@ -3,6 +3,7 @@ package must_gather_test
 import (
 	"testing"
 
+	"github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
```

Comment:
Do these need to be removed as well?

Reply:
Good catch. That's where the tests are failing.